plugins/ShapeLibrary/config.lua

```diff
@@ -0,0 +1,78 @@
+-- The dictionary for shapes
+
+-- If you need extra shapes, just put the shape file (Shape.lua) in
+-- the "Shapes" folder in the plugin directory and add the Category
+-- or shape name in the dictionary
+
+local _M = {
+    [1] = {
+        name = "Geometric-2D",
+        shapes = {
+            [1] = { name = "Arrowhead", shapeName = "arrowHead" }, -- here "name" parameter is seen in dialog as option, so it should be Clearly readable
+            [2] = { name = "Scalene Triangle", shapeName = "scaleneTriangle" },
+            [3] = { name = "Equilateral Triangle", shapeName = "equilateralTriangle" },
+            [4] = { name = "Right Angle Triangle", shapeName = "rightAngleTriangle" },
+            [5] = { name = "Parallelogram", shapeName = "Parallelogram" },
+            [6] = { name = "Hexagon", shapeName = "hexagon" },
+            --  [7] = { name = "My Shape", shapeName = "myShape" },  -- Add your won shape like this
```

Comment:
typo: *won* -> *own*

Reply:
Also maybe we can come up later with a way to avoid the user needing to manually touch this file (probably by writing it programatically, note there is `%q` (https://www.lua.org/manual/5.4/manual.html#pdf-string.format) (drilling into tables still needs to be done manually afaik)

EDIT: Well similar like you do for writing the `giveMeName_and_placeMe_in_shapesFolder.lua`

EDIT2: As this is a reoccurring functionality in this plugin, maybe it makes sense to write a generic functions for this. Something like
```lua
-- writes the lua code representation of x to file
local function write_table_to_config(file, x)
    -- returns the lua code representation of x
    local function write_to_config(x)
        if type(x) == "table" then
            -- TODO analog to below, but with return, not file:write
        else
            return ("%q"):format(x)
        end
    end
    assert(type(x) == "table")
    for k,v in pairs(x) do -- using pairs the ordering in the config won't be deterministic, but the table still contains the keys -> ordering of the resulting table is fine
        -- TODO check if this also works with complex types like tables as keys
        file:write(("[%q] = %q"):format(write_to_config(k), write_to_config(v)))
    end
end
```

Reply:
See https://github.com/MiltonBalaOfficial/ShapeLibraryWithLgi which already does updates without having to touch a file manually.

Reply:
Oh my bad, and there's already a function `serializeTable` (not as generic as I suggested, but suffices for the use-case I guess). Sorry for putting up some noise (just thought the repo would be in sync with this PR because there had been changes recently).